src/library/vtkF3DGenericImporter.h

```diff
@@ -140,7 +140,7 @@ class vtkF3DGenericImporter : public vtkImporter
 
   vtkNew<vtkF3DMetaReader> Reader;
 
-  const F3DOptions* Options = nullptr;
+  const f3d::options* Options = nullptr;
```

Comment:
shared_ptr

Reply:
weak_ptr maybe ?

Reply:
yes indeed

Reply:
I dont see the point though. the ptr will never be deleted during the lifetime of the generic importer.